tests/Unit/ShopifyServiceTest.php

```
@@ -2,424 +2,136 @@
-use GuzzleHttp\Psr7\Response;
 use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
 use Illuminate\Support\Facades\Log;
-use Mockery;
 use PHPUnit\Framework\Attributes\Test;
 use Tests\TestCase;
 
 class ShopifyServiceTest extends TestCase
 {
-    private $mockProductRepository;
+    private $originalShop;
```

PHPMD (2.15.0)

Avoid unused private fields such as '$originalShop'. (undefined)

(UnusedPrivateField)